sdk/eventhub/Microsoft.Azure.WebJobs.Extensions.EventHubs/src/Microsoft.Azure.WebJobs.Extensions.EventHubs.csproj

```diff
@@ -12,18 +12,9 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <!--
-        TEMP:
-            Project reference is needed until the CheckpointStore is included in a release.  Revert when
-            the 5.7.x line is released for GA.
-
-    <PackageReference Include="Azure.Messaging.EventHubs" />
-    -->
-    <ProjectReference Include="$(MSBuildThisFileDirectory)..\..\Azure.Messaging.EventHubs\src\Azure.Messaging.EventHubs.csproj" />
-    <!-- END TEMP -->
+   <PackageReference Include="Azure.Messaging.EventHubs" />
```

Comment:
```suggestion
    <PackageReference Include="Azure.Messaging.EventHubs" />
```

Reply:
Nice catch!   I'll commit this once the test run completes.